Puppet/Puppet.xcodeproj/project.pbxproj

```diff
@@ -788,6 +812,9 @@
 						DevelopmentTeam = L4RARDNZ2Y;
 						ProvisioningStyle = Manual;
 						SystemCapabilities = {
+							com.apple.BackgroundModes = {
```

Comment:
I get why we are doing this, but we need to remember during testing that we can't assume our users set this in their apps.